meta-lmp-support/recipes-bsp/u-boot/u-boot-fio_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend_imx8mm-lpddr4-evk := "${THISDIR}/${PN}:"
 
-SRC_URI_append_imx8mm-lpddr4-evk += "file://0001-Disable-1V8-on-usdhc2-for-SDMUX.patch "
+SRC_URI_append_imx8mm-lpddr4-evk = " file://0001-Disable-1V8-on-usdhc2-for-SDMUX.patch"
```

Comment:
I might understand this wrong, but I would think it is proper to use += to add to _append_xxx variable, so that also other places can add to that append variable which will be appended in later phase.

Reply:
_append is not part of the variable name, but the append operation for SRC_URI variable.